### DISCUSSION TITLE
Filter

### DISCUSSION THREAD

**Discussion by O1egga** [mar10/wunderbaum#92] [General]
Hi, good plugin.
In treegrid can i filter by "Autor", "Year" or other columns?

![1](https://github.com/mar10/wunderbaum/assets/24392193/dadcb376-e2a8-4272-9be2-e6ec5cb0ab12)

**Comment by mar10** [comment]
Yes, this is doable by passing a callback to [tree.filterNodes()](https://mar10.github.io/wunderbaum/api/classes/wunderbaum.Wunderbaum.html#filterNodes).

See also [the documentation](https://mar10.github.io/wunderbaum/#/tutorial/tutorial_filter?id=filter-nodes).
